src/services/dashboard-service.js

```diff
@@ -136,6 +136,34 @@ class DashboardService {
 
     return activities;
   }
+
+  async getUserFavorites(userId) {
```

Comment:
We could make this the user object itself (which is there in `req.user` when calling this service).

Reply:
True, cause the `user` in the controller has the whole data 🤔 
Changed, thanks.